### ISSUE TITLE
[ISSUE for v2]:

### ISSUE THREAD

**Issue by AuraPuno** [w-okada/voice-changer#1581]
### Voice Changer Version

voice-changer-windows-amd64-cuda.zip

### Operational System

Windows 10

### GPU

rtx 3060

### CUDA Version

b2332

### Read carefully and check the options

- [x] If you use win_cuda_torch_cuda edition, setup cuda? [see here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [ ] If you use win_cuda edition, setup cuda and cudnn? [see here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [ ] If you use mac edition, client is not launched automatically. Use chrome to open application.?
- [x] I've tried to change the Chunk Size
- [x] I've tried to set the Index to zero
- [x] I've read the [tutorial](https://github.com/w-okada/voice-changer/blob/master/tutorials/tutorial_rvc_en_latest.md)
- [x] I've tried to extract to another folder (or re-extract) the .zip file

### Does pre-installed model work?

No

### Model Type

RVC

### Issue Description

microphone not working with sup2 enabled

### Application Screenshot

<img width="916" height="935" alt="Image" src="https://github.com/user-attachments/assets/49f5c088-cf52-43d3-a40c-1bdef3a6a08e" />

### Logs on console

2025-10-29 04:44:11,473 INFO     [main] Python: 3.12.7 (tags/v3.12.7:0b05ead, Oct  1 2024, 03:06:41) [MSC v.1941 64 bit (AMD64)]
2025-10-29 04:44:11,474 INFO     [main] Voice changer version: b2332 NVIDIA-CUDA
2025-10-29 04:44:11,474 INFO     [WeightDownloader] Loading weights.
2025-10-29 04:44:11,834 INFO     [Downloader] Verified pretrain/crepe_onnx_full.onnx
2025-10-29 04:44:11,837 INFO     [Downloader] Verified pretrain/crepe_onnx_tiny.onnx
2025-10-29 04:44:11,895 INFO     [Downloader] Verified pretrain/crepe_full.pth
2025-10-29 04:44:11,898 INFO     [Downloader] Verified pretrain/crepe_tiny.pth
2025-10-29 04:44:12,062 INFO     [Downloader] Verified pretrain/content_vec_500.onnx
2025-10-29 04:44:12,142 INFO     [Downloader] Verified pretrain/rmvpe.pt
2025-10-29 04:44:12,363 INFO     [Downloader] Verified pretrain/rmvpe.onnx
2025-10-29 04:44:12,394 INFO     [Downloader] Verified pretrain/fcpe.pt
2025-10-29 04:44:12,426 INFO     [Downloader] Verified pretrain/fcpe.onnx
2025-10-29 04:44:12,426 INFO     [WeightDownloader] All weights are loaded!
2025-10-29 04:44:12,427 INFO     [main] protocol: HTTP
2025-10-29 04:44:14,882 INFO     [loader] Loading faiss with AVX2 support.
2025-10-29 04:44:14,907 INFO     [loader] Successfully loaded faiss with AVX2 support.
2025-10-29 04:44:15,570 INFO     [VoiceChangerManager] Initializing...
2025-10-29 04:44:15,595 INFO     [DeviceManager] Initialized DeviceManager. Backend statuses:
2025-10-29 04:44:15,595 INFO     [DeviceManager] * DirectML: False, device count: 0
2025-10-29 04:44:15,595 INFO     [DeviceManager] * CUDA: True, device count: 1
2025-10-29 04:44:15,596 INFO     [DeviceManager] * MPS: False
2025-10-29 04:44:15,598 INFO     [DeviceManager] Switched to 0: NVIDIA GeForce RTX 3060 (CUDA) (cuda:0). FP16 support: True
2025-10-29 04:44:15,599 INFO     [IORecorder] -------------------------- - - - C:\Users\Vamplure\AppData\Local\Temp\tmp9xe23o4o\tmp_dir\in.wav, C:\Users\Vamplure\AppData\Local\Temp\tmp9xe23o4o\tmp_dir\out.wav
2025-10-29 04:44:15,752 INFO     [VoiceChangerV2] Allocated SOLA buffer size: 4800
2025-10-29 04:44:15,752 INFO     [VoiceChangerManager] Initialized.
2025-10-29 04:44:15,753 INFO     [VoiceChangerManager] Loading RVC...
2025-10-29 04:44:15,753 INFO     [RVCr2] Allocated audio buffer size: 5909
2025-10-29 04:44:15,753 INFO     [RVCr2] Allocated convert buffer size: 14080
2025-10-29 04:44:15,753 INFO     [RVCr2] Allocated pitchf buffer size: 89
2025-10-29 04:44:15,753 INFO     [RVCr2] Initializing...
torch\nn\utils\weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
2025-10-29 04:44:17,021 INFO     [EmbedderManager] Loading embedder hubert_base
2025-10-29 04:44:19,037 INFO     [PitchExtractorManager] Loading pitch extractor rmvpe
voice_changer\common\rmvpe\rmvpe.py:337: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  cpt = torch.load(model_path, map_location=device if device.type == 'cuda' else 'cpu')
2025-10-29 04:44:20,409 INFO     [PipelineGenerator] Loading index...
2025-10-29 04:44:20,409 INFO     [PipelineGenerator] Try loading "model_dir\0\model.index"...
2025-10-29 04:44:20,736 INFO     [Pipeline] GENERATE INFERENCER<voice_changer.RVC.inferencer.RVCInferencerv2.RVCInferencerv2 object at 0x000001BABA5E88C0>
2025-10-29 04:44:20,737 INFO     [Pipeline] GENERATE EMBEDDER<voice_changer.embedder.OnnxContentvec.OnnxContentvec object at 0x000001BABA389490>
2025-10-29 04:44:20,738 INFO     [Pipeline] GENERATE PITCH EXTRACTOR<voice_changer.pitch_extractor.RMVPEPitchExtractor.RMVPEPitchExtractor object at 0x000001BABA463EC0>
2025-10-29 04:44:20,741 INFO     [RVCr2] Initialized.
2025-10-29 04:44:20,741 INFO     [MMVC_Rest] Initializing...
2025-10-29 04:44:20,762 INFO     [MMVC_Rest] Initialized.
2025-10-29 04:44:20,762 INFO     [MMVC_SocketIOApp] Initializing...
2025-10-29 04:44:20,764 INFO     [MMVC_SocketIOApp] Initialized.
2025-10-29 04:44:20,997 INFO     [main] --------
2025-10-29 04:44:20,998 INFO     [main] The server is listening on http://127.0.0.1:18888/
2025-10-29 04:44:20,999 INFO     [main] --------
2025-10-29 04:44:30,938 INFO     [MMVC_Namespace] Connected SID: LsiKiBG3vhWuU91xAAAB
